<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OBSActionsWidget</class>
 <widget class="QWidget" name="OBSActionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>474</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OBS Actions Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_obs_action">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_obs_action">
       <property name="text">
        <string>Section</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_obs_action">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Frontend</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Scenes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sources</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Filters</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Outputs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Encoders</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Services</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_obs_output" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <spacer name="verticalSpacer_top">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>3</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QSplitter" name="splitter_obs_output_scene">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_obs_output_scene">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Scene</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_obs_output_scene">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QSplitter" name="splitter_obs_output_source">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_obs_output_source">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Source</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_obs_output_source">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QSplitter" name="splitter_obs_output_filter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_obs_output_filter">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Filter</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_obs_output_filter">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QSplitter" name="splitter_obs_output_extra_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_obs_output_extra_1">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>extra1</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_obs_output_extra_1">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QSplitter" name="splitter_obs_output_extra_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_obs_output_extra_2">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>extra2</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_obs_output_extra_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QSplitter" name="splitter_obs_output_extra_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_obs_output_extra_3">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>extra3</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_obs_output_extra_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QSplitter" name="splitter_obs_output_extra_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_obs_output_extra_4">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>extra4</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_obs_output_extra_4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QSplitter" name="splitter_obs_output_extra_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_obs_output_extra_5">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>extra5</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_obs_output_extra_5">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QSplitter" name="splitter_obs_output_action">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_obs_output_action">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Action</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_obs_output_action">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item row="10" column="0">
       <spacer name="verticalSpacer_bottom">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>3</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
